packages/blockchain-wallet-v4/src/redux/kvStore/bch/selectors.js

```diff
@@ -14,20 +15,39 @@ export const getAccountsList = state => {
   return lift(a => map(key => a[key], keys(a)))(accountsObj)
 }
 
-export const getContext = (state) => {
+export const getContext = (state, spendable) => {
```

Comment:
what do you need spendable for?

Reply:
This is a remnant of something I was trying out. Will fix